goss-testing/tests/ncn/goss-ip-dns-check.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2014-2021 Hewlett Packard Enterprise Development LP
+command:
+  ip_dns_check:
+    title: Check for dupe and DNS entries for NCN and Uans
+    meta:
+      desc: Checks for dupe IPs in SMD EthernetInterface table and NCN/UAN management network DNS entries resolve to only 1 IP.
```

Comment:
might be good to add some info here on how to re-run the script for more verbose output, or indicate what to try if the test fails?